R/fetch_metal_pdb.R

```diff
@@ -0,0 +1,240 @@
+#' Fetch information about structure protein-metal binding from MetalPDB
```

Comment:
That sounds weird, do you mean "fetch structural information about protein-metal binding (...)" ?